changelogs/unreleased/memtx-stmt-rollback-oom.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/memtx
+
+* Fixed possible crash/panic on hitting memtx memory OOM.
```

Comment:
Missing issue reference.

"hitting memtx memory out-of-memory" sounds weird.

Reply:
There is no issue for this commit. However there is a ticket for similar issue in memcs - https://github.com/tarantool/tarantool-ee/issues/1107.

Reworded to to `Fixed possible crash/panic on hitting memtx memory limit`.